src/lang_pt-br/lang_text.h

```diff
@@ -10,25 +10,167 @@
 
 // == #define overrides for language strings go here
 
+#undef INTRO_DIALOG_WIN_Y
+#define INTRO_DIALOG_WIN_Y DIALOG_TEXT_HEIGHT_LINES(16)
 #undef __INTRO_MESSAGE_STR
 #define __INTRO_MESSAGE_STR \
 /*  |------------------| Available space (18 chars wide) */\
-    "WELCOME TO WORDYL \n" \
+    "BEMVINDO AO WORDYL\n" \
     "\n" \
-    "LANGUAGE ... " STR(LANG_CODE) "\n" \
-    "  A..Z  ONLY\n" \
-    "  NO SPECIAL CHARS\n" \
+    "IDIOMA ... " STR(LANG_CODE) "\n" \
+    "  A..Z  APENAS\n" \
+    "  SEM CAR. ESPEC.\n" \
     "\n" \
-    "IN GAME MENU:\n" \
+    "ABRIR MENU:\n" \
     "  3 x SELECT\n" \
     "\n" \
-    "BOARD CURSOR:\n" \
-    "  SELECT + B A\n" \
+    "SELECIONAR:\n" \
```

Comment:
If it's possible to fit then ideally this would be more specific that it refers to Select + A or B moving the cursor on the game board (compared with normal selection on the input keyboard).

Reply:
Oh, yes, I missed the right context. It should be more clear now. Cursor has the same meaning here, so we shouldn't translate that. I think adding "Mover" makes more sense and fits the space. Now it reads like: "to move the cursor press...".

Reply:
Closer maybe, but I think this still misses the "Game Board" aspect, which is important to emphasize. 

If "Cursor" has the same meaning, is there a short word which can reference the "Game Board" (or the "Board Tiles" / "Game Tiles" which make up the board) and combine that with Cursor?

As a rough example (no idea if they are suitable), some google translate:
* cursor do tabuleiro (1 letter too long, but might be `Cursor do\n Tabuleiro:Sel + AB`)
* cursor de ladrilho
* cursor de telhas

Hope you don't mind my being particular on this. :)

Reply:
"Cursor dos Blocos" makes more sense and is shorter. It's a translation from "Tiles Cursor"
"Tabuleiro" works too, but we use it more for physical board games.
![Captura de tela de 2022-04-08 20-24-48](https://user-images.githubusercontent.com/5665358/162545980-c4c78f4d-9709-4cfa-9c1b-f18294e3a186.png)

Reply:
Perfect :+1: